### PR TITLE
node autoscaler changes for prod

### PR DESCRIPTION
**Pull request by avi-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: templates/vars/aks/prod.json [hunk 1]
```diff
@@ -3,22 +3,22 @@
...
     "enableAutoScaling": {
-      "value": ""
+      "value": "true"
     },
     "type": {
-      "value": "AvailabilitySet"
+      "value": "VirtualMachineScaleSets"
     },
     "vmSize": {
-      "value": "Standard_D4s_v3"
```
Comment (naikajah) [inline templates/vars/aks/prod.json:21]: same here why are we changing the vmsize?

Comment (avi-hmcts) [reply templates/vars/aks/prod.json:21]: yeah it's corrected now

File: templates/vars/aks/prod.json [hunk 2]
```diff
@@ -3,19 +3,19 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "kubernetesVersion": {
-      "value": "1.17.9"
+      "value": "1.18.8"
     },
     "nodeMinCount": {
       "value": 32
```
Comment (naikajah) [inline templates/vars/aks/prod.json:9]: Should we have min count as 32?

**Review by naikajah — APPROVED** [review]

File: templates/vars/aks/prod.json (hunk 2 above)
Comment (avi-hmcts) [reply templates/vars/aks/prod.json]: changed it min:30 and max:50 as per our discussion

**Pull request merged**